src/services/app/user.ts

```diff
@@ -493,18 +497,45 @@ export default class UserService {
         }
         // until here
 
+        let response: IUserHello = {
+            user: {
+                suspect: user.suspect,
+            },
+        };
+
+        if (manager) {
+            response = {
+                ...response,
+                manager: {
+                    communityId: community ? community.id : undefined,
+                },
+            };
+        }
+
+        if (beneficiary) {
+            response = {
+                ...response,
+                beneficiary: {
+                    blocked: beneficiary !== null ? beneficiary.blocked : false,
+                    readRules: readBeneficiaryRules,
+                    communityId: community ? community.id : undefined,
+                },
```

Comment:
nah, you don't need all that. My idea was really to make it simpler. Just return it as it is (same for others). Make sure you query just that on the queries above instead of everything. For managers for example you get only the communityId. This community id is now the publicId, but no worries, there are other tasks to replace this. We will keep using the deprecated value in UI in the meantime.
```suggestion
                beneficiary,
```

Reply:
also, the `getCommunity` will later be removed.